pkg/networkmanager/interfaces.js

```diff
@@ -1445,6 +1445,18 @@ PageNetworking.prototype = {
         $("#networking-add-bridge").click($.proxy(this, "add_bridge"));
         $("#networking-add-vlan").click($.proxy(this, "add_vlan"));
 
+        // HACK - hide "Add Team" if it doesn't work due to missing
```

Comment:
> Why a line break for "bits"?

That's what Emacs does by default, no?  It breaks lines after column 70.

Fixed.